src/loop_closure_assistant.cpp

```diff
@@ -144,19 +144,28 @@ void LoopClosureAssistant::publishGraph()
 /*****************************************************************************/
 {
   interactive_server_->clear();
-  std::unordered_map<int, Eigen::Vector3d> * graph = solver_->getGraph();
+  auto graph = solver_->getGraph();
 
   if (graph->size() == 0) {
     return;
   }
 
-  RCLCPP_DEBUG(node_->get_logger(), "Graph size: %i", (int)graph->size());
+  RCLCPP_DEBUG(node_->get_logger(), "Graph size: %zu", graph->size());
   bool interactive_mode = false;
   {
     boost::mutex::scoped_lock lock(interactive_mutex_);
     interactive_mode = interactive_mode_;
   }
 
+  const auto& vertices = mapper_->GetGraph()->GetVertices();
```

Comment:
Linting: should be spaces, use `ament_cpplint` and `ament_uncrustify` to resolve inconsistencies with your changes